### PR TITLE
Beabranch

### PR DESCRIPTION
**Pull request by LibbyKnight**

### CODE REVIEW

File: secondPage.html [hunk 1]
```diff
@@ -105,7 +105,10 @@ <h3 id="movieTitle">No Title</h3>
 	
 						<div class="small-9 columns">
 
-							<input type="number" id="middle-label" placeholder="12345">
+
+						<form action="javascript:0">
```
Comment (dzimmerman902) [inline secondPage.html]: Does the form need a closing element? </form>

Comment (LibbyKnight) [inline secondPage.html]: hmm it was there. tho it works without it lol. ill add it and push again

**Comment by AdamJahur** [comment]
okay looks good to me

**Comment by dzimmerman902** [comment]
I'll merge for now but please be aware of how you write the code. Yes it works but the indentation is off and need re-aligning.

**Pull request merged**